scripts/get_bank_registry_be.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+import json
+import xlrd
+import requests
+
+URL = "https://www.nbb.be/doc/be/be/protocol/r_fulllist_of_codes_current.xlsx"
+
+
+def process():
+    registry = []
+    skip_names = ["NAV", "VRIJ", "NAP", "NYA", "VRIJ - LIBRE", "-"]
+    book = xlrd.open_workbook(file_contents=requests.get(URL).content)
+
+    try:
+        sheet = book.sheet_by_index(0)
+        for row in list(sheet.get_rows())[2:]:
+            bank_code, bic, name, second_name = row[:4]
+            if bic.value.upper() in skip_names:
+                continue
+            if name.value != "":
+                registry.append({
+                    "country_code": "BE",
+                    "primary": True,
+                    "bic": bic.value.upper().replace(" ", ""),
+                    "bank_code": bank_code.value,
+                    "name": name.value,
+                    "short_name": name.value,
+                })
+            else:
+                registry.append({
+                    "country_code": "BE",
+                    "primary": True,
+                    "bic": bic.value.upper().replace(" ", ""),
+                    "bank_code": bank_code.value,
+                    "name": second_name.value,
+                    "short_name": second_name.value,
+                })
+        return registry
+    except IndexError:
```

Comment:
What exactly does it help to catch an exception here only to re-raise it right away? Additionally we're talking about a script here that you run once in a while: If the structure of the document changes you have to adapt the script anyway. This is not an application that has to deal with all kind of error situations.

Reply:
Based on your first remarks, I was under the impression you wanted me to make it more robust. I suppose I was mislead by that remark and felt I had to figure out a way. I'm sorry.